scripts/local-deploy.sh

```diff
@@ -32,10 +56,16 @@ cd $repo_basedir;
 echo
 echo -e "\e[1;33mDeploying osparc on ${MACHINE_FQDN}, using credentials $SERVICES_USER:$SERVICES_PASSWORD...\e[0m"
 
+
+# -------------------------------- PORTAINER ------------------------------
 echo
 echo -e "\e[1;33mstarting portainer...\e[0m"
-pushd ${repo_basedir}/services/portainer; make up; popd
+pushd ${repo_basedir}/services/portainer
+sed -i "s/PORTAINER_ADMIN_PWD=.*/PORTAINER_ADMIN_PWD=$SERVICES_PASSWORD/" .env
```

Comment:
will not work in mac ... define ``sed`` variable and replace with ``gsed`` if OSX (Darwin)

As a side note: I think compatibility with OSX is so far easy to support ... nonetheless, if it becomes a burden we can also drop it

Reply:
@ignapas is creating another PR for Mac.

Did not know Mac was a requirement... more than windows? why? ;)

Reply:
oh because it's cooler, Windows is cheap...